recipes/openbabel/2.4.0/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: openbabel
+  version: 2.3.90dev7d621d9
```

Comment:
Shame on me... That probably wasn't saved by Atom editor which is still open at work. Nice catch!